apps/frontend/src/pages/MyProfile.tsx

```diff
@@ -0,0 +1,28 @@
+import { Box, Typography, useMediaQuery, useTheme } from "@mui/material";
+import AccountSidebar from "../components/AccountSidebar";
+import SearchBar from "../components/searchBar";
+
+const MyProfile = () => {
+  const theme = useTheme();
+  const isDesktop = useMediaQuery(theme.breakpoints.up("sm"));
+
+  return (
+    <>
+      <SearchBar />
```

Comment:
Adding the <SearchBar/> here is breaking the Account pages. Can we add the <SearchBar/> component to the AccountSidebar component and remove `SearchBar` from MyListings, MyProfile, and MyReviews pages. I tested it and it looks like it works.

Reply:
Something like this for the AccountSidebar.tsx file:

```
import {
  Box,
  Divider,
  Drawer,
  List,
  ListItem,
  Typography,
} from "@mui/material";
import { useNavigate, useParams } from "react-router-dom";
import { useStyles } from "../styles/pageStyles";
import AccountSidebarItem from "./AccountSidebarItem";
import SearchBar from "./searchBar";

const AccountSidebar = ({ selectedItem }: { selectedItem: string }) => {
  const styles = useStyles();
  const navigate = useNavigate();
  const { id } = useParams();

  return (
    <>
      <SearchBar />
      <Box sx={{ display: "flex" }}>
        <Drawer variant="permanent" sx={styles.drawer}>
          <Divider />
          <List>
            <ListItem onClick={() => navigate(`/user/${id}`)}>
              <Typography variant="h4" sx={{ fontWeight: "bold" }}>
                Account
              </Typography>
            </ListItem>
            <Divider />
            <AccountSidebarItem
              path={`/user/${id}`}
              itemName="My Profile"
              selected={selectedItem === "My Profile"}
            />
            <Divider />
            <AccountSidebarItem
              path="/user/listings"
              itemName="My Listings"
              selected={selectedItem === "My Listings"}
            />
            <Divider />
            <AccountSidebarItem
              path="/user/reviews"
              itemName="My Reviews"
              selected={selectedItem === "My Reviews"}
            />
            <Divider />
          </List>
        </Drawer>
      </Box>
    </>
  );
};

export default AccountSidebar;

```

Reply:
Updated